designsafe/apps/rapid/openTopo_Data/scripts/centerpoint_Requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests==2.31.0
```

Comment:
an empty __init__.py is needed at `designsafe/apps/rapid/openTopo_Data/scripts/` and  at `designsafe/apps/rapid/openTopo_Data/`